src/test/resources/metaModel/proto/addressBook/district.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package address_book.district;
+
+import "custom_message_options.proto";
+import "custom_field_options.proto";
+
+enum TestEnum {
+   TEST_ENUM_VALUE_1 = 0;
+   TEST_ENUM_VALUE_2 = 1;
+   TEST_ENUM_VALUE_3 = 2;
+}
+
+message AddressBookDistrict {
+   option (custom_options.ctx) = cluster;
+   string name = 1 [(custom_field_options.key) = true, Test single inline option];
```

Comment:
The second option should be a valid option from `custom_field_options.proto`. Use the new `internal` option.